modules/installation-vsphere-infrastructure.adoc

```diff
@@ -52,34 +52,32 @@ You must install the {product-title} cluster on a VMware vSphere version 7 insta
 [cols=2, options="header"]
 |===
 |Virtual environment product |Required version
-|VM hardware version | 15 or later
+|VMware virtual hardware | 15 or later
 |vSphere ESXi hosts | 7.0.2 or later
 |vCenter host   | 7.0.2 or later
 |===
 
 [IMPORTANT]
 ====
-Installing a cluster on VMware vSphere version 7.0.1 or earlier is now deprecated. These versions are still fully supported, but version 4.11 of {product-title} requires vSphere virtual hardware version 15 or later. Hardware version 15 is now the default for vSphere virtual machines in {product-title}. To update the hardware version for your vSphere nodes, see the "Updating hardware on nodes running in vSphere" article.
-
-If your vSphere nodes are below hardware version 15 or your VMware vSphere version is earlier than 6.7.3, upgrading from {product-title} 4.10 to {product-title} 4.11 is not available.
+Installing a cluster on VMware vSphere versions 7.0.0 and 7.0.1 is deprecated. These versions are still fully supported, but all vSphere 6.x versions are no longer supported. Version 4.12 of {product-title} requires VMware virtual hardware version 15 or later. To update the hardware version for your vSphere virtual machines, see the "Updating hardware on nodes running in vSphere" article in the _Updating clusters_ section.
```

Comment:
4.13 only supports 7.0u2 or higher. So a cluster on 4.12 which is on 7.0.0 or 7.0.1 can't be upgraded to 4.13 and hence versioning requirements has to be tightened further.

Reply:
@bscott-rh, please open a separate PR against main/4.13 to capture this change.

Reply:
@gnufied, it'll be better from a process standpoint if we merge this PR and open a follow-up.